src/formik.tsx

```diff
@@ -91,14 +95,20 @@ export interface FormikState<V> {
 export interface FormikActions<P, V> {
   /* Manually set top level error */
   setError: (e: any) => void;
-  /* Manually set Errors */
+  /* Manually set errors object */
   setErrors: (errors: FormikErrors) => void;
   /* Manually set isSubmitting */
   setSubmitting: (isSubmitting: boolean) => void;
-  /* Manually set touched fields */
+  /* Manually set touched object */
   setTouched: (touched: FormikTouched) => void;
-  /* Manually set values  */
+  /* Manually set values object  */
   setValues: (values: V) => void;
+  /* Set value of form field directly */
+  setFieldValue: (field: string, value: any) => void;
```

Comment:
I don't mind the idea of `setField()` as an additional option, as long as `setFieldValue()`, `setFieldTouched()`, and `setFieldError()` are still also available. I think you still need the latter three because `setField()`'s signature would be so specific to Formik and it's a little more cumbersome to use if you're just setting one thing.

Let's definitely not give up and add `setFormikState()` yet -- that pushes a ton of inappropriate responsibility to the caller. We may need it as an escape hatch at some point but it shouldn't be the blessed way to do anything.